pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -98,6 +100,28 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	logger.WithField("targetNamespaces", targetNamespaces).Debug("updated target namespaces")
 
 	if namespacesChanged(targetNamespaces, op.Status.Namespaces) {
+		ogLabel := getOperatorGroupLabel(*op)
+		// Remove labels from targetNamespaces that were dropped.
+		for _, namespaceName := range unique(logger, op.Status.Namespaces, targetNamespaces) {
+			err = a.removeNamespaceLabel(namespaceName, ogLabel)
+			if err != nil {
+				logger.WithError(err).Warnf("failed to remove operatorgroup label from namespace %s", namespaceName)
+				return err
+			}
+		}
+
+		// Add labels to targetNamespaces that were added.
+		unique := unique(logger, targetNamespaces, op.Status.Namespaces)
```

Comment:
super nit: could we use a better name for `unique` that won't require the reader to scroll to and read the func?

Reply:
Going with Ben's suggestion: https://github.com/operator-framework/operator-lifecycle-manager/pull/1398#discussion_r395353177

Reply:
Resolved.